### PR TITLE
Feat(eos_cli_config_gen): Add support for Interface Vxlan, vxlan bridging vtep-to-vtep

### LINKED ISSUE #3702: EOS_CLI_Config_gen support for Internface Vxlan vxlan bridging vtep-to-vtep
### Enhancement summary

For customers using WiFi Vxlan Anchor feature the switch needs the following missing command from eos_cli_config_gen

interface vxlan1
 vxlan bridging vtep-to-vtep

### Which component of AVD is impacted

eos_cli_config_gen

### Use case example

Customers using Arista Wifi with Vxlan Anchor feature for an SSID require a specific Vxlan configuration on the Anchor switch.  The current structured config is missing one line and EOS_CLI is needed as a workaround.  

interface vxlan1
 vxlan source-interface loopback1
 vxlan virtual-router encapsulation mac-address mlag-system-id
 vxlan udp-port 4789
 **vxlan bridging vtep-to-vtep**
 vxlan flood vtep learned data-plane
 vxlan vlan 50 vni 50

### Describe the solution you would like

support for vxlan bridging vtep-to-vtep under interface Vxlan

### Describe alternatives you have considered

current workaround is:    eos_cli: 'vxlan bridging vtep-to-vtep'

### Additional context

_No response_

### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by laxmikantchintakindi**
## Change Summary

For customers using WiFi Vxlan Anchor feature the switch needs vxlan bridging vtep-to-vtep command in eos_cli_config_gen.

## Related Issue(s)

Fixes #3702

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes
Add key for bridging in vxlan-interface schema.
Add the cli in templates.

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/vxlan_interface.schema.yml [hunk 1]
```diff
@@ -37,6 +37,8 @@ keys:
                 type: int
                 convert_types:
                   - str
+              vtep_to_vtep_bridging:
+                type: bool
```
Comment (Vibhu-gslab) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/vxlan_interface.schema.yml:41]: Add description.

Comment (laxmikantchintakindi) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/vxlan_interface.schema.yml:41]: Added.

**Review by MaheshGSLAB — APPROVED** [review]
LGTM, tested on lab with custom_structured_configuration

**Review by gmuloc — APPROVED** [review]
LGTM

**Pull request merged**